tests/unit/components/home/infoSideBar/TermsTable.spec.js

```diff
@@ -1,5 +1,5 @@
 import { mount } from "@vue/test-utils";
-import TermsTable from "@/components/home/infoSideBar/TermsTable.vue";
+import TermsTable from "../../../../../src/components/home/directory/viewer/TermsTable.vue";
```

Comment:
use @/components

Reply:
Had an issue with @ where if I were to move the file it points to, VSCode did not pick up the change and did not update the path. With ../ changes were picked up and paths updated on by VSCode.